.zenodo.json

```diff
@@ -0,0 +1,83 @@
+{
+  "description": "DeepReg is a freely available, community-supported open-source toolkit for research and education in medical image registration using deep learning.",
+  "license": "Apache-2.0",
+  "title": "DeepReg: a deep learning toolkit for medical image registration",
+  "upload_type": "software",
+  "keywords": [
+    "Deep Learning",
+    "Image Fusion",
+    "Medical Image Registration",
+    "Neural Networks"
+  ],
+  "creators": [
+    {
+      "orcid": "0000-0002-1184-7421",
+      "name": "Fu, Yunguan"
+    },
+    {
+      "orcid": "0000-0001-5685-971X",
+      "name": "Brown, Nina Monta\u00f1a"
+    },
+    {
+      "orcid": "0000-0002-5004-0663",
+      "name": "Saeed, Shaheer U."
+    },
+    {
+      "orcid": "0000-0002-0539-3638",
+      "name": "Casamitjana, Adri\u00e0"
+    },
+    {
+      "orcid": "0000-0001-6838-335X",
+      "name": "Baum, Zachary M. C."
```

Comment:
@zacbaum Can you verify your name and maybe also check others ;)

Reply:
Mine is good now, thanks